utility/make_rbd_storage_class.py

```diff
@@ -0,0 +1,57 @@
+import os
+from ocs.utils import create_oc_resource
+from utility import templating
+from utility.get_ceph_secret import get_ceph_secret
+from utility.get_ip_addrs import get_ip_addrs_for_ceph_type
+
+
+def make_rbd_pool(cluster_namespace, local_dir):
+    """
+    Make an rbd pool for the calling rbd storage class
+
+    Args:
+        cluster_namespace (str): namespace used by tester.
```

Comment:
cluster_namespace (str): **namespace used by tester**. Can it be better to describe this as the "namespace of the rook ceph cluster"  or something equivalent. 

normally namespace used by tester implies a new project created by tester to run his app pods and create pvcs.